### PR TITLE
Create trapped-on-an-island-with-a-pig.md

### PR DESCRIPTION
**Pull request by DrTeo**
Welcome to Pig island (The new reality TV show)

2nd Draft.

### CODE REVIEW

File: content/english/anti-vegan/trapped-on-an-island-with-a-pig.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        name: valerie-redfern
+        role: co-author
+    - 
+        name: sam-martin
+        role: editor
+    -
+        name: deb-hardwicke
+        role: proofreader
+draft: false
+description: "If it's just you and a pig on a desert island surely you would eat the pig to survive?"
```
Comment (Sam-Martin) [inline content/english/anti-vegan/trapped-on-an-island-with-a-pig.md]: Needs to be prefixed with `How to respond to the argument:` so it matches the others :)

File: content/english/anti-vegan/trapped-on-an-island-with-a-pig.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+weight: 5
+---
+# Argument
+
+If it's just you and a pig on a desert island surely you would eat the pig to survive?
```
Comment (Sam-Martin) [inline content/english/anti-vegan/trapped-on-an-island-with-a-pig.md]: Can we make this a block quote with `>` so it matches the others too?

File: content/english/anti-vegan/trapped-on-an-island-with-a-pig.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+# Rebuttal
+
+## Quick Responses
+
+> That’s a difficult question. If it's just you and a child on a desert island  would you eat the child to  survive? 
+
+or
+
+> That’s an interesting thought experiment but I think you have misunderstood the implications. It is designed to test the Ethics of Emergencies and whether they differ in scope from everyday ethics. That the survivors of Flight 571 turned to cannabalism to survive in an emergency was not a mandate for cannabalism under everyday conditions. Similarly, even if a vegan did kill and eat the pig in an emergency this does not justify the unnecessary killing of animals under normal conditions.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/trapped-on-an-island-with-a-pig.md]: Shall we swap `I think you've misunderstood the implications` to something like `but let's be clear that this doesn't help us understand whether it's okay to eat animals in normal circumstances`. Less combative?

File: content/english/anti-vegan/trapped-on-an-island-with-a-pig.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+## Detail - Analytic
+
+### How does the argument go?
+
+This thought experiment is usually presented as some sort of *“Gotcha”*. 
+
+> If it's just you and a pig on a desert island surely you would eat the pig so you could survive yourself? Then you wouldn't be vegan anymore would you?" *(snigger)*.  
+> Followed by: So if you (the vegan) choose to eat the pig, then that's no different from me eating pork when I choose.
+
+This line of reasoning is both fallacious (as discussed shortly) and shows a misunderstanding of the scope and nature of the thought experiment:
```
Comment (Sam-Martin) [inline content/english/anti-vegan/trapped-on-an-island-with-a-pig.md]: Candidate for simplification? I don't think `fallacious` is a common term unfortunately :(

File: content/english/anti-vegan/trapped-on-an-island-with-a-pig.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+- This couldn’t happen because… *(numerous reasons are presented)*. Yes, the thought experiment presents an improbable scenario but one that is logically possible. However, it really does not matter if the event could occur. Thought experiments are artificial constructs designed to test specific moral principles; you might argue that all thought experiments are invalid but it is far more interesting to engage and consider the implications.
+- You can’t kill the humans; that’s murder!  (Perhaps, but let’s not confuse legality with morality. *Is* the action morally justified? *…and remember you will never be charged with murder.*)
+
+### What’s the answer?
+
+The temptation with thought experiments is to rush to an answer and then be forced to turn cartwheels in an attempt to justify that answer. Consider: 
+
+- Would you kill the animal but never the child? Why? Just because they are human, but without a specific reason or via some claim to human superiority?
```
Comment (Sam-Martin) [inline content/english/anti-vegan/trapped-on-an-island-with-a-pig.md]: > Just because they are human, but without a specific reason or via some claim to human superiority? 

This is a little hard to parse. Consider rewording?

File: content/english/anti-vegan/trapped-on-an-island-with-a-pig.md (hunk 1 above)
Comment (DrTeo) [reply content/english/anti-vegan/trapped-on-an-island-with-a-pig.md]: Done

File: content/english/anti-vegan/trapped-on-an-island-with-a-pig.md (hunk 2 above)
Comment (DrTeo) [reply content/english/anti-vegan/trapped-on-an-island-with-a-pig.md]: Done

File: content/english/anti-vegan/trapped-on-an-island-with-a-pig.md (hunk 3 above)
Comment (DrTeo) [reply content/english/anti-vegan/trapped-on-an-island-with-a-pig.md]: K. Took combative part out. Already ends much like suggested or do you want it shortened?

File: content/english/anti-vegan/trapped-on-an-island-with-a-pig.md (hunk 4 above)
Comment (DrTeo) [reply content/english/anti-vegan/trapped-on-an-island-with-a-pig.md]: Probably right (Though Val didn't trip on it)

File: content/english/anti-vegan/trapped-on-an-island-with-a-pig.md (hunk 5 above)
Comment (DrTeo) [reply content/english/anti-vegan/trapped-on-an-island-with-a-pig.md]: Would you kill the animal but not a child? Is that *just* because they are human? Are you making a claim of [human superiority]({{< ref "/anti-vegan/humans-are-superior" >}})?

Now just to add citations

File: content/english/anti-vegan/trapped-on-an-island-with-a-pig.md [hunk 6]
```diff
@@ -16,12 +16,12 @@ contributors:
         name: deb-hardwicke
         role: proofreader
 draft: false
-description: "If it's just you and a pig on a desert island surely you would eat the pig to survive?"
+description: "How to respond to the argumnent: If it's just you and a pig on a desert island surely you would eat the pig to survive?"
```
Comment (Sam-Martin) [inline content/english/anti-vegan/trapped-on-an-island-with-a-pig.md]: Typo: `argumnent`

File: content/english/anti-vegan/trapped-on-an-island-with-a-pig.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+contributors:
+    - 
+        name: tery-hardwicke
+        role: author
+    - 
+        name: valerie-redfern
+        role: co-author
+    - 
+        name: sam-martin
+        role: editor
```
Comment (Sam-Martin) [inline content/english/anti-vegan/trapped-on-an-island-with-a-pig.md]: Bump me down to proofreader on this one! I didn't do much!

File: content/english/anti-vegan/trapped-on-an-island-with-a-pig.md (hunk 6 above)
Comment (DrTeo) [reply content/english/anti-vegan/trapped-on-an-island-with-a-pig.md]: Yeah noticed that 15 seconds after hitting commit. DoH!

**Review by Sam-Martin — APPROVED** [review]
Awesome! 👍
